### ISSUE TITLE
Unable to start the app even after reinstall (request failed, insufficient permission)

### ISSUE THREAD

**Issue by mchsk** [abhishekbanthia/Clocker#47]
`xxxxxxx-macbook:~ xxx$ /Applications/Clocker.app/Contents/MacOS/Clocker`
->
```
Initializing menubar timer
2018-09-06 19:04:46.862 Clocker[892:16558] CoreData: XPC: Unable to load metadata: Error Domain=NSCocoaErrorDomain Code=134070 "An error occurred in the persistent store." UserInfo={Problem=request failed, insufficient permission}
CoreData: error: -addPersistentStoreWithType:NSXPCStore configuration:(null) URL:file:///Users/mchsk/Library/Calendars/Calendar%20Cache options:{
    NSInferMappingModelAutomaticallyOption = 1;
    NSMigratePersistentStoresAutomaticallyOption = 1;
    agentOrDaemon = 1;
    serviceName = "com.apple.CalendarAgent.database";
} ... returned error Error Domain=NSCocoaErrorDomain Code=134070 "An error occurred in the persistent store." UserInfo={Problem=request failed, insufficient permission} with userInfo dictionary {
    Problem = "request failed, insufficient permission";
}

Refetching events from the store
2018-09-06 19:04:46.882 Clocker[892:16558] [Crashlytics] Version 3.8.5 (123)
2018-09-06 19:04:46.882 Clocker[892:16558] [Crashlytics] Running on MacBookPro14,1, 10.13.6 (17G65)
2018-09-06 19:04:46.888 Clocker[892:16558] [Answers] Initialized
2018-09-06 19:04:46.889 Clocker[892:16558] [Fabric] Initialized with kit versions: {
    "com.twitter.answers.mac" = "1.3.5";
    "com.twitter.crashlytics.mac" = "3.8.5";
    "io.fabric.sdk.mac" = "1.6.1";
}
2018-09-06 19:04:46.989 Clocker[892:16593] [Answers] Answers enabled with settings: {
    "flush_interval_secs" = 300;
    "flush_on_background" = 1;
    "forward_to_google_analytics" = 0;
    "include_purchase_events_in_forwarded_events" = 0;
    "max_byte_size_per_file" = 8000;
    "max_file_count_per_send" = 1;
    "max_pending_send_file_count" = 100;
    "sampling_rate" = 1;
    "track_custom_events" = 1;
    "track_predefined_events" = 1;
    "track_view_controllers" = 0;
    url = "https://e.crashlytics.com/spi/v2/events";
}
2018-09-06 19:04:46.989 Clocker[892:16593] [Answers] Logging events to /Users/mchsk/Library/Containers/com.abhishek.Clocker/Data/Library/Caches/com.crashlytics.data/com.abhishek.Clocker/analytics/v2/events
2018-09-06 19:04:47.074 Clocker[892:16604] [Answers] Transmitting packaged events for '344A8FF6-A747-49B8-9F96-516E8C00D28A'.
2018-09-06 19:04:47.197 Clocker[892:16620] [Fabric] settings downloaded successfully
2018-09-06 19:04:47.197 Clocker[892:16620] [Fabric] Settings are available for consumption
2018-09-06 19:04:47.368 Clocker[892:16620] [Answers] Finished transmission of '344A8FF6-A747-49B8-9F96-516E8C00D28A'.
```

**Comment by mchsk** [comment]
Hey, so as far as I remember there were some settings in the Clocker where I have checked two checkboxes for the permissions (?) - I cant remember precisely since the app cannot be open anymore.
Since then it could not be open, nothing visible in the toolbar but using `htop` I saw it as a process.

I tried removing some Calendar cache files ->
https://wilkinsit.ca/mac-osx/clear-calendar-cache-mac-os/

but did not help :) I have `NO IDEA WHAT TO DO` but i ❤️ the app!! 👍

**Comment by abhishekbanthia** [comment]
Thanks for reporting. Can you try resetting the defaults and verifying if the app opens up? Here's how you do that:

1. Close Clocker (Force quit the process from the Activity Monitor if neccesary!)
2. Go to Terminal and type in the following command: `defaults delete com.abhishek.Clocker`
3. Press the Return key
4. Open Clocker again!

**Comment by mchsk** [comment]
@abhishekbanthia hey, seems to have worked thanks 👍

**Comment by abhishekbanthia** [comment]
Great. The issue should be fixed in the 1.6.04!
